.github/workflows/cross-version.yml

```diff
@@ -23,6 +23,22 @@ jobs:
     container: "junitteam/build:${{ matrix.jdk }}"
     steps:
       - uses: actions/checkout@v2
+      - name: Cache Gradle wrapper and dependencies
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.gradle/caches/
+            ~/.gradle/wrapper/dists
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
```

Comment:
Shouldn't this also include `buildSrc/**`, `gradle/**`, and `gradle.properties`?

Reply:
Good point, I've expanded the list.